### ISSUE TITLE
Duplicate entries...

### ISSUE THREAD

**Issue by jothomp** [MicrosoftDocs/OfficeDocs-Enterprise#130]
It seems that ID11 and ID21 are identical. Any reason?

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 077406d8-bff1-496e-3ea2-b38558fd9355
* Version Independent ID: 57db3dec-2250-2dee-fd1a-ec6656911798
* Content: [Office 365 URLs and IP address ranges](https://docs.microsoft.com/en-gb/office365/enterprise/urls-and-ip-address-ranges?redirectSourcePath=%252fen-us%252farticle%252fOffice-365-URLs-and-IP-address-ranges-8548a211-3fe7-47cb-abb1-355ea5aa88a2#sharepoint-online-and-onedrive-for-business)
* Content Source: [Enterprise/urls-and-ip-address-ranges.md](https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/blob/live/Enterprise/urls-and-ip-address-ranges.md)
* Service: **o365-administration**
* GitHub Login: @JoeDavies-MSFT
* Microsoft Alias: **josephd**

**Comment by AndreaBarr** [comment]
@jothomp Thank you for this feedback.

I want to make sure this duplicated information gets looked into so I am going to get this over to the Enterprise team for investigation. 

I was able to get this into the backlog for the Enterprise team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
